recipe/meta.yaml

```
@@ -34,13 +32,68 @@ test:
     - udunits2 -h
     - udunits2 -H meter -W miles
 
+outputs:
+  - name: libudunits2
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    files:
+      - lib/libudunits2.so.*  # [linux]
```

```suggestion
      - lib/libudunits2.so*  # [linux]
```
It looks like `lib/libudunits2.so` is ending up in the `udunits2` package, not in `libudunits2`.

---

It does, but that's correct. When another package is built against Libudunits2, it will use `libudunits2.so` to find the latest version of the library. But as a requirement, it will record the concrete version of the shared object, c.f. the output of `ldd libnco.so`:
```
libudunits2.so.0 => /home/a002160/mambaforge/envs/evt/lib/././libudunits2.so.0 (0x00007fd4cb5d5000)
```

That's why `libudunits2.so` is needed only at build time, whereas all the `libudunits2.*` are needed at run time.

---

Thanks for the explanation.